app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx

```diff
@@ -35,12 +29,15 @@ const StationActionButtons = () => {
         >
           {t("calendarHeader.copyLink")} <img src={copyLink} />
         </Button>
-        <Button
+        {
+          // In MVP we don't have this feature
+          /* <Button
           onClick={() => {}}
           aria-labelledby={t("calendarHeader.altExportSession")}
         >
           {t("calendarHeader.exportSession")} <img src={downloadImage} />
-        </Button>
+        </Button> */
+        }
```

Comment:
I'd normally be removing such pieces, my take is that we hope to be doing the post MVP work, but I am not 100% sure. :)

Reply:
Done